From 790fb17f3a866016c897f1c903ca9eeb8a4b693c Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@gmail.com>
Date: Wed, 9 Feb 2022 21:14:37 +0200
Subject: [PATCH 4/5] Bump Security String to 2022-02-05

Implemented:
============
None

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2020-13112  A-194342672    EoP    High       10, 11                  5122fd5
CVE-2020-13113  A-196085005    EoP    High       10, 11                  d0c470b, 2cb0c0b
CVE-2021-0706   A-193444889    DoS    High       10, 11                  3d2b22cd4c0b
CVE-2021-39619  A-197399948    EoP    High       11, 12                  ba7e4213040f, dc06656820b1
CVE-2021-39631  A-193890833    ID     High       10, 11, 12              3fb89e3728, 86e355fba3
CVE-2021-39662  A-197302116    EoP    High       11, 12                  ee937df1
CVE-2021-39666  A-204445255    ID     High       11, 12                  78390aadaf, 8c5a4e5c46
CVE-2021-39668  A-193445603    EoP    High       11, 12                  95205d4a112a
CVE-2021-39669  A-196969991    EoP    High       11, 12                  3daf3bec2d
CVE-2021-39674  A-201083442    EoP    High       10, 11, 12              de6e37829 (aosp), 701d743d20 (caf)
CVE-2021-39676  A-197228210    EoP    High       11                      b7957a6b56bd

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2021-39663  A-200682135    EoP    High       10
CVE-2021-39664  A-203938029    ID     High       12
CVE-2021-39665  A-204077881    ID     High       12
CVE-2021-39671  A-206718630    EoP    High       12
CVE-2021-39675  A-205729183    EoP    Critical   12

Change-Id: I8acaebd9df31015b691649ecb3837ec03ec37d78
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 9e1b85992..9c0a59fab 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2022-01-05
+      PLATFORM_SECURITY_PATCH := 2022-02-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.25.1

